### PR TITLE
[System] Add internal APIs for power off and reboot

### PR DESCRIPTION
**Pull request by lokilee73**
### Description of Change ###
Add internal APIs for power off and reboot

### API Changes ###
 - System.Power.PowerOff
 - System.Power.Reboot

### CODE REVIEW

**Review by hyotaekshim — APPROVED** [review]
Looks good.

File: src/Tizen.System/Device/Power.cs [hunk 1]
```diff
@@ -154,5 +155,24 @@ public static void ReleaseLock(PowerLock type)
                 throw DeviceExceptionFactory.CreateException(res, "unable to release power lock.");
             }
         }
+        /// <summary>
+        /// Power off the device.
+        /// </summary>
+        /// <exception cref="UnauthorizedAccessException">If the privilege is not set.</exception>
+        /// <exception cref="InvalidOperationException">In case of any system error.</exception>
+        /// <example>
```
Comment (myroot) [inline src/Tizen.System/Device/Power.cs:163]: It must be need a special privilege, but not mentioned which privilege is need,
Please add <privilege> description

Comment (lokilee73) [reply src/Tizen.System/Device/Power.cs:164]: Applied the request

**Review by hyotaekshim — APPROVED** [review]
Looks good.

**Review by WonyoungChoi — APPROVED** [review]

**Pull request merged**